spec/models/unit_spec.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: units
+#
+#  id              :bigint           not null, primary key
+#  name            :string           not null
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint
+#
+require "rails_helper"
```

Comment:
Isn't this auto-required?

Reply:
Yes, though it's required in tons of these specs. I'm removing it from here and separately we can bulk-remove it.

Reply:
I created https://github.com/rubyforgood/human-essentials/pull/4529 to get the others